package.json

```diff
@@ -8,10 +8,14 @@
     "@rails/ujs": "^6.0.0-alpha",
     "@rails/webpacker": "^4.2.2",
     "arctic_admin": "3.2.0",
+    "serialize-javascript": "^4.0.0",
```

Comment:
can we have it as a "resolution" instead of a direct dependency?

Reply:
Okey, great!